securemailbox/constants.py

```diff
@@ -0,0 +1,4 @@
+# Declare any variables (as constants) that are or could be used anywhere in the application
+
+# TODO: Determine what a good fingerprint length is
+FINGERPRINT_LENGTH = 100
```

Comment:
Nit: Github complains with no newline at EOF.
```suggestion
FINGERPRINT_LENGTH = 100

```

Reply:
Oh my bad.  Idk how I missed that but I'll fix irt.